From 67e8f7da8a4ca23202e3a10e6c37c7ac886fd1aa Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Thu, 28 Feb 2019 23:59:53 +0000
Subject: [PATCH 5/8] Add C2D_FORMAT_BT601_FULLRANGE to C2D_FORMAT_MODE

* Another definition that CAF forgot to include in public HALs,
  leading to libc2dcolorconvert target breakage.

Change-Id: Ida86923bcf9b8298a728bd0dcd08cf0410b2a8c9
---
 libcopybit/c2d2.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libcopybit/c2d2.h b/libcopybit/c2d2.h
index 821a6d577..2ee71b1ac 100644
--- a/libcopybit/c2d2.h
+++ b/libcopybit/c2d2.h
@@ -72,6 +72,7 @@ typedef enum {
     C2D_FORMAT_MACROTILED        = (1 << 16), /* tiled in macro level */
     C2D_FORMAT_TILED_4x4         = (1 << 17), /* 4x4 tiled format */
     C2D_FORMAT_SWAP_RB           = (1 << 18), /* Swap R & B color components */
+    C2D_FORMAT_BT601_FULLRANGE   = (1 << 20), /* YUV format is full range following BT 601 */
     C2D_FORMAT_UBWC_COMPRESSED   = (1 << 23), /* UBWC compressed format */
 } C2D_FORMAT_MODE;
 
-- 
2.34.1

